app/src/main/java/fr/cdudit/playgroundfinder/features/map/MapFragment.kt

```diff
@@ -79,4 +106,15 @@ class MapFragment : Fragment(), GoogleMap.OnMarkerClickListener {
         }
         return false
     }
+
+    override fun onLocationChanged(p0: Location) {
+        mapFragment?.getMapAsync { map ->
+            this.actualPosition?.remove()
+            this.actualPosition = map.addMarker(
+                MarkerOptions()
+                    .position(LatLng(p0.latitude, p0.longitude))
+                    .icon(BitmapDescriptorFactory.fromResource(R.drawable.round_map))
```

Comment:
Le `R.drawable.round_map` doit être récupéré depuis le `viewModel`, quelque chose comme ça `this.viewModel.getIconRoundMapId()` et dans le viewModel :
```kotlin
@ResId
fun getIconRoundMapId(): Int = R.drawable.round_map
```